configd/src/apps/sentinel/env.cpp

```diff
@@ -9,9 +9,12 @@
 
 LOG_SETUP(".env");
 
+using namespace std::chrono_literals;
+
 namespace config::sentinel {
 
-constexpr std::chrono::milliseconds CONFIG_TIMEOUT_MS(3 * 60 * 1000);
+constexpr std::chrono::milliseconds CONFIG_TIMEOUT_MS = 3min;
+constexpr std::chrono::milliseconds MODEL_TIMEOUT_MS = 1500ms;
```

Comment:
consider using a bit more timeout

Reply:
already did that :-)
I really wanted < 500ms to ensure this doesn't impact anything if the model config is unavailable.  Since this happens after the first config subscription is OK it should be fast anyway.